lib/netsparker/vulnerability.rb

```diff
@@ -128,13 +128,14 @@ def cleanup_html(source)
       result.gsub!(/<i>(.*?)<\/i>/) { "_#{$1}_" }
       result.gsub!(/<em>(.*?)<\/em>/) { "*#{$1}*" }
       result.gsub!(/<h2>(.*?)<\/h2>/) { "*#{$1}*" }
-      result.gsub!(/<strong>(.*?)<\/strong>/) { "*#{$1}*" }
+      result.gsub!(/<strong>(.*?)<\/strong>/) { "*#{$1}* " }
 
-      result.gsub!(/(<br>)|(<br\/>)/, "\n")
+      result.gsub!(/(<br>)|(<br(\s*)\/>)/, "\n")
       result.gsub!(/(<div>)|(<\/div>)/, "")
       result.gsub!(/<font.*?>(.*?)<\/font>/m, '\1')
       result.gsub!(/<p (.*?)>(.*?)<\/p>/) { "\n#{$2}\n" }
       result.gsub!(/<span(.*?)>(.*?)<\/span>/, '\2')
+      result.gsub!(/<span(.*?)>|<\/span>/, "")
```

Comment:
If we have this, do we need the line above it? I can't think of a use for the first line now.

Reply:
Without both lines, the same file I provided fails. If you can come up with a better system that doesn’t leave span tags, please do!

Reply:
*sample file